### PR TITLE
vpn-policy-routing: update to 0.3.4

### PR DESCRIPTION
**Pull request by stangri**
Maintainer: me
Compile tested: x86_64, Sophos SG-105, 21.02-SNAPSHOT r15926
Run tested:  x86_64, Sophos SG-105, 21.02-SNAPSHOT r15926, start/stop, ifdown/ifup one interface, verify iptables rules

Description: this includes the following updates:
* remove create/remove_lock functions as they are not needed with PROCD
* rename config file and update Makefile
* introduce `reload_interface` function and `extra_command` to allow reloading of only affected interface
* remove unneeded code from `service_triggers`

Signed-off-by: Stan Grishin <stangri@melmac.net>

### CODE REVIEW

File: net/vpn-policy-routing/Makefile [hunk 1]
```diff
@@ -4,8 +4,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vpn-policy-routing
-PKG_VERSION:=0.3.2
-PKG_RELEASE:=20
+PKG_VERSION:=0.3.4
+PKG_RELEASE:=0
```
Comment (aparcar) [inline net/vpn-policy-routing/Makefile:8]: I always get told that releases should start with `1`.

Comment (stangri) [reply net/vpn-policy-routing/Makefile:8]: Thanks, I'll push a fix within a few days.

Comment (aparcar) [reply net/vpn-policy-routing/Makefile:8]: Ping?

Comment (stangri) [reply net/vpn-policy-routing/Makefile:8]: Sorry,  an issue was discovered and I didn't have the time to look into during the week. If I can't push the update by tomorrow I'll close these.

Comment (aparcar) [reply net/vpn-policy-routing/Makefile:8]: No rush, I just thought you forgot about this.

**Comment by stangri** [comment]
Can the i386 build failure be safely ignored?

**Comment by stangri** [comment]
I'll need to work a bit more on this, closing this PR for now.

**Pull request closed without merging**